src/industry_gui.cpp

```diff
@@ -732,6 +736,7 @@ class BuildIndustryWindow : public Window {
 	void OnHundredthTick() override
 	{
 		if (_game_mode == GM_EDITOR) return;
+		if (count == 0) return;
```

Comment:
```suggestion
		if (this->count == 0) return;
```
Code style guidelines prescribe that class members must always be qualified,